shared/common-adapters/name-with-icon/index.tsx

```diff
@@ -129,7 +129,10 @@ class NameWithIcon extends React.Component<NameWithIconProps> {
         colorFollowing={this.props.colorFollowing}
         colorYou={this.props.notFollowingColorOverride || true}
         notFollowingColorOverride={this.props.notFollowingColorOverride}
-        style={this.props.size === 'smaller' ? {} : styles.fullWidthText}
+        style={Styles.collapseStyles([
+          styles.textBold,
```

Comment:
This is necessary even with the `type="BodyBold"` above?

Reply:
yeah, because the type has this: 
`type={this.props.horizontal ? 'BodyBold' : adapterProps.titleType}`
and titleType can be a lot of things. 
In fact let's remove that statement and only leave `adapterProps.titleType`, then simply use the style attribute for bolding.